guillotina/contrib/mailer/__init__.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from guillotina import configure
+from guillotina.interfaces import IMailer
+from guillotina.component import provide_utility
+from guillotina.utils import import_class
+import logging
+
+logger = logging.getLogger('guillotina.contrib.mailer')
+
+app_settings = {
+    "mailer": {
+        "default_sender": "foo@bar.com",
```

Comment:
Should we leave `deault_sender` to None and log warning if not provided and trying to send?

Reply:
agree